### DISCUSSION TITLE
Radar Diagram

### DISCUSSION THREAD

**Discussion by imsergiobernal** [mermaid-js/mermaid#6150] [Ideas]
Straight to the point: I would add radar diagram. Is anyone open to help? I have never contributed here.

![image](https://www.datanovia.com/en/wp-content/uploads/2020/12/radar-chart-in-r-customized-fmstb-radar-chart-1.png)

**Comment by imsergiobernal** [comment]
https://github.com/imsergiobernal/mermaid/tree/develop

**Comment by StephenWalrond** [comment]
I came here looking for this diagram. @imsergiobernal Has this idea got any traction?

**Reply by knsv** [reply]
It is included in mermaid 11.6.0

**Reply by knsv** [reply]
🎉
